### ISSUE TITLE
Using the guides on nextjs.org no longer work

### ISSUE THREAD

**Issue by raelmiu** [zeit/next.js#11942]
# Bug report

## Describe the bug

Setting up a new project following this guide: https://nextjs.org/learn/basics/getting-started/setup
Results in the following error: 
```

> next-test2@1.0.0 dev /Users/jesperbylund/Projects/next-test2
> next

[ wait ]  starting the development server ...
[ info ]  waiting on http://localhost:3000 ...
Error: Your custom PostCSS configuration must export a `plugins` key.
    at getPostCssPlugins (/Users/jesperbylund/Projects/next-test2/node_modules/next/dist/build/webpack/config/blocks/css/plugins.js:6:79)
    at async __overrideCssConfiguration (/Users/jesperbylund/Projects/next-test2/node_modules/next/dist/build/webpack/config/blocks/css/overrideCssConfiguration.js:1:272)
    at async getBaseWebpackConfig (/Users/jesperbylund/Projects/next-test2/node_modules/next/dist/build/webpack-config.js:101:120)
    at async Promise.all (index 0)
    at async HotReloader.start (/Users/jesperbylund/Projects/next-test2/node_modules/next/dist/server/hot-reloader.js:14:1788)
    at async DevServer.prepare (/Users/jesperbylund/Projects/next-test2/node_modules/next/dist/server/next-dev-server.js:10:1565)
    at async /Users/jesperbylund/Projects/next-test2/node_modules/next/dist/cli/next-dev.js:21:359
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! next-test2@1.0.0 dev: `next`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the next-test2@1.0.0 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

```
## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Go to https://nextjs.org/learn/basics/getting-started/setup
2. Follow the steps in the guide
3. Run `npm run dev`

## Expected behavior

That it should start next.

## System information

- OS: macOS 10.15.3
- Version of Next.js: 
```
{
  "name": "next-test2",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "dev": "next",
    "build": "next build",
    "start": "next start"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "dependencies": {
    "next": "^9.3.5",
    "react": "^16.13.1",
    "react-dom": "^16.13.1"
  }
}

```
- Version of Node.js: v13.8.0

**Comment by timneutkens** [comment]
Looks like you have a `postcss.config.js` somewhere in the directory tree above your application 🤔

**Comment by raelmiu** [comment]
Wow, you are absolutely correct. I must have cloned a project into the wrong folder some time ago.

Thank you, it all works as expected now.

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
